peachjam/models/core_document_model.py

```diff
@@ -407,6 +424,21 @@ class Meta:
     def __str__(self):
         return f"{self.doc_type} - {self.title}"
 
+    def apply_labels(self):
+        """Apply labels to this document based on its properties."""
+        # label showing that a document is cited/reported in law reports, hence "more important"
+        label, _ = Label.objects.get_or_create(
```

Comment:
The concept of "reported" only applies to judgments, so this should all move to the judgment object.

So `CoreDocument.apply_labels()` is a no-op and doesn't do anything.